File: 2785-sort-vowels-in-a-string/2785-sort-vowels-in-a-string.py
class Solution:
    def sortVowels(self, s: str) -> str:
        st =['a','e','i','o','u','A','E','I','O','U']
        s=list(s)
        re=[]
        for i in range(len(s)):
            if s[i] in st:
                re.append(s[i])
        re.sort()
        i=0
        for j in range(len(s)):
            if s[j] in st:
                s[j]=re[i]
                i+=1
        print(s)
        return "".join(s)
                

File: 2785-sort-vowels-in-a-string/2785-sort-vowels-in-a-string.cpp
class Solution {
public:
    string sortVowels(string s) {
        vector<char> vowels;
        string vowelSet = "aeiouAEIOU";
        int n = s.length();
        for(int i=0;i<n;i++){
            if(vowelSet.find(s[i]) != string::npos)
                vowels.push_back(s[i]);
        }
        
        sort(vowels.begin(), vowels.end());
        int idx = 0;
    
        for(int i=0;i<n;i++){
            if(vowelSet.find(s[i]) != string::npos)
                s[i] = vowels[idx++];
        }
        return s;


    }
};